homeassistant/components/elmax/common.py

```diff
@@ -137,21 +147,25 @@ def __init__(
         coordinator: ElmaxCoordinator,
     ) -> None:
         """Construct the object."""
+        super().__init__(coordinator=coordinator)
         self._panel = panel
         self._device = elmax_device
         self._panel_version = panel_version
-        self._coordinator = coordinator
-        self._transitory_state = None
+        self._client = coordinator.http_client
+        self._transitory_state: Any = None
+        # Keep a local reference to the coordinator to better deal with type checking ad avoid
+        # casting.
+        self._elmax_coordinator = coordinator
 
     @property
-    def transitory_state(self) -> Any | None:
-        """Return the transitory state for this entity."""
-        return self._transitory_state
+    def elmax_coordinator(self) -> ElmaxCoordinator:
+        """Return the Elmax coordinator."""
+        return self._elmax_coordinator
 
-    @transitory_state.setter
-    def transitory_state(self, value: Any) -> None:
-        """Set the transitory state value."""
-        self._transitory_state = value
+    @property
+    def elmax_client(self):
```

Comment:
We don't need this method, as all subclasses can already access the protected client.

Reply:
That is true. Yet, in this case it's a matter of style. I like to limit the usage of private member "_member" into subclasses. 
You might argue that is a protected field not a private, and you'd be right. 

Let me know if is ok or if refactoring in this case is necessary.

Reply:
A single underscore is protected, not private, so... It is incorrect at this point already. Let's avoid proxy methods like these. There is not need to add complexity.

Reply:
Actually, one might argue that a _single underscore_ stands literally for [non-public](https://docs.python.org/3/tutorial/classes.html#private-variables) while _mangling_ is just an anti-collision trick.

In any case, that's ok for me, I got rid of the proxy as you asked.